components/edit-collective/sections/UserTwoFactorAuth.js

```diff
@@ -147,6 +149,9 @@ class UserTwoFactorAuth extends React.Component {
           })
           .then(data => {
             this.setState({ recoveryCodes: get(data, 'data.addTwoFactorAuthTokenToIndividual.recoveryCodes') });
+          })
+          .then(() => {
+            this.props.refetchLoggedInUser(); // No need to await
```

Comment:
An alternative to this pattern would be to add the named query `LoggedInUser` to the refecthQueries option of the mutation.

Reply:
True, I went with consistency since the helper was already plugged into the component and it's what we're using in most places to refresh the logged-in account.